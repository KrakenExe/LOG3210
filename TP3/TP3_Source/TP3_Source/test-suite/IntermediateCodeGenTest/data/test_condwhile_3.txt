Declaration
num a;
num b;
bool c;
num d;
EndDeclaration

a = d;
while(!c) {
  a = a + 42;
  b = 1 / 3 + 5;
}
b = a;    
