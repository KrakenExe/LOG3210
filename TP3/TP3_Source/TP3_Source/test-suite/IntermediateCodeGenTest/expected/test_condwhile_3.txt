a = d
_L1
_L3
if c == 1 goto _L2
goto _L4
_L4
_t0 = a + 42
a = _t0
_L5
_t2 = 1 / 3
_t1 = _t2 + 5
b = _t1
goto _L3
_L2
b = a
_L0
